code/renderers/svelte/template/cli/ts-4-9/Button.stories.ts

```diff
@@ -1,5 +1,4 @@
 import type { Meta, StoryObj } from '@storybook/svelte';
-
```

Comment:
is this missing the onClick fn like the other templates are using?

Reply:
We have decided to not yet invest time in this for Svelte, as Svelte 5 will change the way events are handled in svelte.